src/Avalonia.Controls.ColorPicker/Themes/Fluent/ColorView.xaml

```diff
@@ -0,0 +1,609 @@
+      4.54514 11.3796C4.33279 11.3691 4.00262 11.2625 3.47218 10.8301C3.0866 10.5158 2.90473
+      10.0668 2.90237 9.48908C2.89995 8.89865 3.08843 8.20165 3.42324 7.47971C4.09686 6.0272
+      5.28471 4.63649 6.29249 4.01905Z
+    </PathGeometry>
+    <!-- Fluent UI System Icons : ic_fluent_options_20_regular.svg -->
+    <PathGeometry x:Key="ColorViewComponentsIconGeometry">
+      M14.95 5C14.7184 3.85888 13.7095 3 12.5 3C11.2905 3 10.2816 3.85888 10.05 5H2.5C2.22386
+      5 2 5.22386 2 5.5C2 5.77614 2.22386 6 2.5 6H10.05C10.2816 7.14112 11.2905 8 12.5 8C13.7297
+      8 14.752 7.11217 14.961 5.94254C14.9575 5.96177 14.9539 5.98093 14.95 6H17.5C17.7761 6 18
+      5.77614 18 5.5C18 5.22386 17.7761 5 17.5 5H14.95ZM12.5 7C11.6716 7 11 6.32843 11 5.5C11
+      4.67157 11.6716 4 12.5 4C13.3284 4 14 4.67157 14 5.5C14 6.32843 13.3284 7 12.5 7ZM9.94999
+      14C9.71836 12.8589 8.70948 12 7.5 12C6.29052 12 5.28164 12.8589 5.05001 14H2.5C2.22386
+      14 2 14.2239 2 14.5C2 14.7761 2.22386 15 2.5 15H5.05001C5.28164 16.1411 6.29052 17 7.5
+      17C8.70948 17 9.71836 16.1411 9.94999 15H17.5C17.7761 15 18 14.7761 18 14.5C18 14.2239
+      17.7761 14 17.5 14H9.94999ZM7.5 16C6.67157 16 6 15.3284 6 14.5C6 13.6716 6.67157 13 7.5
+      13C8.32843 13 9 13.6716 9 14.5C9 15.3284 8.32843 16 7.5 16Z
+    </PathGeometry>
+  </Styles.Resources>
+
+  <Style Selector="ColorView">
+    <Setter Property="CornerRadius" Value="{DynamicResource ControlCornerRadius}" />
+    <Setter Property="Template">
+      <ControlTemplate>
+        <Grid RowDefinitions="Auto,Auto">
+
+          <!-- Set the backgrounds -->
+          <Border Grid.Row="0"
+                  Grid.RowSpan="2"
+                  Height="48"
+                  HorizontalAlignment="Stretch"
+                  VerticalAlignment="Top"
+                  Background="{DynamicResource SystemControlBackgroundBaseLowBrush}"
+                  CornerRadius="{TemplateBinding CornerRadius}" />
+
+          <!-- Available for Fluent v2 use -->
+          <!--<Border Grid.Row="0"
+                  Grid.RowSpan="2"
+                  HorizontalAlignment="Stretch"
+                  VerticalAlignment="Stretch"
+                  Background="{TemplateBinding Background}" />
+          <Border Grid.Row="0"
+                  Grid.RowSpan="2"
+                  Margin="0,48,0,0"
+                  HorizontalAlignment="Stretch"
+                  VerticalAlignment="Stretch"
+                  Background="{ThemeResource LayerFillColorAltBrush}"
+                  BorderBrush="{ThemeResource DividerStrokeColorDefaultBrush}"
+                  BorderThickness="0,1,0,0" />-->
+
+          <TabControl x:Name="PART_TabControl"
+                      Grid.Row="0"
+                      Height="338"
+                      Width="350"
+                      SelectedIndex="{Binding SelectedTabIndex, RelativeSource={RelativeSource TemplatedParent}, Mode=TwoWay}">
```

Comment:
Note here that if externally the SelectedTabIndex is set to 4 or something (outside the valid range) the TabControl will ignore it and replace it with a valid SelectedIndex. This however is not propogated back through the TwoWay binding so the SelectedTabIndex and SelectedIndex become out of sync.

The work-around for this is done in code-behind since I already have a ValidateSelectedTab() method. However, I'm not sure if the binding system should support this (UWP always had issues like this, WPF perhaps not).

Reply:
I just documented this in code, marking resolved.